CHANGELOG.md

```diff
@@ -6,41 +6,45 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## \[Unreleased]
+
 ### Added
 
-- \[SDK\] A `DeferredTqdmProgressReporter` class, which doesn't have glitchy output
-  like `TqdmProgressReporter` in certain circumstances
+- \[SDK\] Introduced the `DeferredTqdmProgressReporter` class,
+  which avoids the glitchy output seen with the `TqdmProgressReporter` under certain circumstances
   (<https://github.com/opencv/cvat/pull/6556>)
-- \[SDK, CLI\] A `cvat_sdk.auto_annotation` module that provides
-  functionality for automatically annotating a task by running a
-  user-provided function on the local machine, and a corresponding CLI command
-  (`auto-annotate`)
+- \[SDK, CLI\] Added the `cvat_sdk.auto_annotation` module,
+  providing functionality to automatically annotate tasks
+  by executing a user-provided function on the local machine.
+  A corresponding CLI command (`auto-annotate`) is also available
   (<https://github.com/opencv/cvat/pull/6483>)
-- Cached frames indication on the interface (<https://github.com/opencv/cvat/pull/6586>)
+- Included an indication for cached frames in the interface
+  (<https://github.com/opencv/cvat/pull/6586>)
+- Added predefined functions using torchvision,
+  which will serve as a replacement for YOLOv8(<https://github.com/opencv/cvat/pull/6649>)
 
 ### Changed
 
-- Increased default guide assets limitations (30 assets, up to 10Mb each)
+- Raised the default guide assets limitations to 30 assets,
+  with a maximum size of 10Mb each
```

Comment:
```suggestion
  with a maximum size of 10MB each
```

I looked at the linked PR, and it is megabytes, not megabits. Might as well fix it now.